pyinjective/core/ibc/channel/grpc/ibc_channel_grpc_api.py

```diff
@@ -1,6 +1,6 @@
 from typing import Any, Callable, Dict, List, Optional
 
-from grpc._cython.cygrpc import Channel
+from grpc import Channel
```

Comment:
I guess this is the same, only using a non private name. Or is it a different module?

Reply:
Yes, in the end it is the same. I just feel it is better to use the general class